Assets/Scripts/Game Controllers/LevelEndTriggerController.cs

```diff
@@ -3,11 +3,35 @@
 
 public class LevelEndTriggerController : MonoBehaviour
 {
+    public int nextSceneLoad,
```

Comment:
`[SerializeField] private` should be used instead of `public` here (also for `lvlButtons` in `LevelSelection.cs`). It's good to restrict variable visibility so that there are fewer potential possibilities to consider when looking for variable usage.